src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -959,6 +966,8 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             }
             else if (point.Properties().IsRightButtonPressed())
             {
+                Focus(FocusState::Pointer);

```

Comment:
I forgot about the middle click, good call! I tested touch, and it seems to focus anyway (maybe because we already explicitly focus the terminal when we get a `Tap` event). In any case, I've moved the Focusing part of the code to even before we check for `PointerDeviceTypes`, so that any `PointerPressed` events will focus the terminal.